ui/inputhandler.cpp

```diff
@@ -110,24 +110,24 @@ bool InputHandler::eventFilter(QObject *watched, QEvent *event)
 
 void InputHandler::handleMouseMoveEvent(QMouseEvent *event)
 {
-    MouseEventDTO* eventDto = calculateMouseEventDto(event);   
```

Comment:
This and the other `MouseEventDTO`s were all being leaked, so changed to smart pointer.